### PR TITLE
Update public document

### PR DESCRIPTION
**Pull request by kumasumit-msft**
Update public document to fix the existing documentation issue. The mdatp service restart needed to update the "cloud-diagnostic" or "log-rotation-parameters config"

### CODE REVIEW

**Comment by kumasumit-msft** [comment]
@ideeptibhardwaj @mramitarora

File: microsoft-365/security/defender-endpoint/linux-preferences.md [hunk 1]
```diff
@@ -572,7 +572,7 @@ To verify that your /etc/opt/microsoft/mdatp/managed/mdatp_managed.json is worki
 - automatic_definition_update_enabled
 
 > [!NOTE]
-> For the mdatp_managed.json to take effect, no restart of the `mdatp` deamon is required.
+> For the mdatp_managed.json to take effect, restart of the `mdatp` deamon is required only in cases when `cloud-diagnostic` or `log-rotation-parameters` configs are updated.
```
Comment (ideeptibhardwaj) [inline microsoft-365/security/defender-endpoint/linux-preferences.md]: Reword as follows:

No restart of mdatp deamon is required for changes to **most** configs in mdatp_managed.json to take effect.
**Exception**: Following configs will require deamon restart to take effect:
- cloud-diagnostic
- log-rotation-parameters

**Review by ideeptibhardwaj — CHANGES_REQUESTED** [review]

File: microsoft-365/security/defender-endpoint/linux-preferences.md (hunk 1 above)
Comment (kumasumit-msft) [reply microsoft-365/security/defender-endpoint/linux-preferences.md]: Done

File: microsoft-365/security/defender-endpoint/linux-preferences.md [hunk 2]
```diff
@@ -572,8 +572,10 @@ To verify that your /etc/opt/microsoft/mdatp/managed/mdatp_managed.json is worki
 - automatic_definition_update_enabled
 
 > [!NOTE]
-> For the mdatp_managed.json to take effect, no restart of the `mdatp` deamon is required.
-  
+> No restart of mdatp deamon is required for changes to most configs in mdatp_managed.json to take effect.
+  **Exception:** Following configs will require deamon restart to take effect:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/linux-preferences.md]:
```suggestion
> No restart of mdatp daemon is required for changes to _most_ configurations in mdatp_managed.json to take effect.
  **Exception:** The following configurations require a daemon restart to take effect:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kumasumit-msft, kindly commit the suggested changes. Many thanks.

**Pull request merged**